mappings/net/minecraft/tag/GlobalTagAccessor.mapping

```diff
@@ -3,11 +3,25 @@ CLASS net/minecraft/class_5120 net/minecraft/tag/GlobalTagAccessor
 	COMMENT tag changes through reloads/server tag sends.
 	FIELD field_23682 currentContainer Lnet/minecraft/class_5414;
 	FIELD field_23683 tags Ljava/util/List;
+	FIELD field_25740 managerGetter Ljava/util/function/Function;
+	METHOD <init> (Ljava/util/function/Function;)V
+		ARG 1 managerGetter
 	METHOD method_26770 getContainer ()Lnet/minecraft/class_5414;
 	METHOD method_26771 setContainer (Lnet/minecraft/class_5415;)V
+		ARG 1 manager
+	METHOD method_26772 (Lnet/minecraft/class_5414;Lnet/minecraft/class_5120$class_5121;)V
+		ARG 1 tag
 	METHOD method_26773 get (Ljava/lang/String;)Lnet/minecraft/class_3494$class_5123;
 		ARG 1 id
 	METHOD method_27061 markReady ()V
+	METHOD method_27062 (Lnet/minecraft/class_3494;Lnet/minecraft/class_5120$class_5121;)V
+		ARG 1 tag
+	METHOD method_27063 (Lnet/minecraft/class_3494;Lnet/minecraft/class_2960;)Lnet/minecraft/class_3494;
+		ARG 1 id
+	METHOD method_29224 getTagDifferences (Lnet/minecraft/class_5415;)Ljava/util/Set;
```

Comment:
This is really only reporting the elements that the required accessors has and the registry tag manager doesn't have. so this is like `getMissingTags`